db/migrate/20210325112605_add_action_link_fields_to_coronavirus_sub_sections.rb

```diff
@@ -0,0 +1,8 @@
+class AddActionLinkFieldsToCoronavirusSubSections < ActiveRecord::Migration[6.0]
+  def change
+    change_table :coronavirus_sub_sections, bulk: true do |t|
+      t.string :action_link_content
+      t.string :action_link_summary
```

Comment:
I think this should be type `text` as `string` is only 255 characters, and that might not be enough.

Reply:
In the current MVP UI mock up on the [trello](https://trello.com/c/B7f0bkef/257-mvp-create-new-input-fields-for-action-link-content-publishing-tool) card, this field is shown having a 150 character limit - hence it was originally created as a `string` type. The character count was queried with the designer who confirmed that the character limit is not required for the MVP, but seemed un-certain that it wouldn't be needed for the production version.

As it is tricky to convert from a `text` to a `string` type - without loosing data - but the reverse is straightforward, and as there is uncertainty regarding this field - I felt it best to leave this as a `string` for the MVP.

Reply:
Can you add that explanation to the commit message? Thanks!